src/utils.cpp

```diff
@@ -0,0 +1,9 @@
+#include "utils.h"
+
+namespace rerun_vrs {
+
+    std::string add_quotes(const std::string& str) {
+        return "\"" + str + "\"";
```

Comment:
this should ideally also escape any already existing quotes in the string

Reply:
it's a bit annoying though that even we think we need such a util in the example. Should be part of the sdk then
* Related to https://github.com/rerun-io/rerun/issues/3995